mesheryctl/pkg/utils/platform_test.go

```diff
@@ -20,32 +20,6 @@ import (
 
 // The function are related to download should be test in meshkit package, please do not add test here.
 
-func TestPlatform(t *testing.T) {
-	t.Run("GetManifestTreeURL", func(t *testing.T) {
-		_, err := GetManifestTreeURL("master")
-		if err != nil {
-			t.Fatal(err)
-		}
-	})
-}
-
-func TestGetManifestURL(t *testing.T) {
-	t.Run("GetManifestURL with empty manifest", func(t *testing.T) {
-		manifest := Manifest{
-			Typ:  "blob",
-			Path: "testdata/manifest.yaml",
-		}
-		rawManifestsURL := "https://raw.githubusercontent.com/meshery/meshery/master/mesheryctl/pkg/utils/"
-
-		manifestURL := GetManifestURL(manifest, rawManifestsURL)
-		expectedURL := "https://raw.githubusercontent.com/meshery/meshery/master/mesheryctl/pkg/utils/testdata/manifest.yaml"
-
-		if manifestURL != expectedURL {
-			t.Errorf("GetManifestURL failed: expected %s, but got %s", expectedURL, manifestURL)
-		}
```

Comment:
@souvikinator just curious. Any reason why these tests are being removed? Is it because they are failing due to the changes made?

Reply:
@souvikinator need your response here... 👀

Reply:
@alphaX86 
- Removed the `TestPlatform` test as it was failing regardless of changes due to GitHub API limitations. Since we are using GitHub Walker from MeshKit, it's no longer needed.
- Removed `TestGetManifestURL` since the concerned function was no longer in use. It can be added back if needed.

Reply:
@souvikinator noted. PR has been approved from my end.